admin-console/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
```

Comment:
Does this mean just the blue dot over the icon will show when Alerts are firing? Should we update this text to include if there is an available update?

Reply:
There's an open visual design story for the specifics of this. I plan to update once that's fully decided

Reply:
Sounds good!